src/main/java/RedisSessionManagerT8.java

```diff
@@ -0,0 +1,298 @@
+    public void add(Session session) {
+        try {
+            Jedis jedis = redisConnectionPool.getResource();
+            saveToRedis(jedis, session);
+            redisConnectionPool.returnResourceObject(jedis);
+        } catch (IOException ex) {
+            log.warn("Unable to add to session manager store: " + ex.getMessage());
+            throw new RuntimeException("Unable to add to session manager store.", ex);
+        }
+    }
+
+    @Override
+    public Session findSession(String id) throws IOException {
+        Session session = null;
+
+        //Get connection to redis
+        Jedis jedis = redisConnectionPool.getResource();
+
+        Hashtable<String, Object> metadata =
+                (Hashtable)SerializationUtils.deserialize(Base64.getDecoder().decode(jedis.get((id + REDIS_METADATA_KEY).getBytes())));
+
+        Hashtable<String, Object> attributes =
+                (Hashtable)SerializationUtils.deserialize(Base64.getDecoder().decode(jedis.get((id + REDIS_ATTRIBUTES_KEY).getBytes())));
+
+        session = getSession(metadata, attributes);
+
+        redisConnectionPool.returnResourceObject(jedis);
+
+        return session;
+    }
+
+    @Override
+    public void remove(Session session) {
+
+        remove(session, false);
+    }
+
+    @Override
+    public void remove(Session session, boolean update) {
+        //Get connection to redis
+        Jedis jedis = redisConnectionPool.getResource();
+
+        //Just remove the attributes not the metadata
+        jedis.del((session.getId() + REDIS_ATTRIBUTES_KEY).getBytes());
+
+        redisConnectionPool.returnResourceObject(jedis);
+    }
+
+    @Override
+    public void processExpires() {
+       //We need to implement the expire process
+    }
+
+    //------------------------------------------------------------------------------------------------------------------
+    // END Sessions
+    //------------------------------------------------------------------------------------------------------------------
+
+
+    //------------------------------------------------------------------------------------------------------------------
+    // START Utils
+    //------------------------------------------------------------------------------------------------------------------
+
+    /**
+     * Get session from metadata and attributes
+     * @param metadata
+     * @param attributes
+     * @return
+     */
+    private StandardSession getSession(Hashtable<String, Object> metadata, Hashtable<String, Object> attributes){
+        StandardSession standardSession = (StandardSession) createEmptySession();
+        standardSession.setValid(Boolean.valueOf((String)metadata.get(METADATA_VALID)));
+        SimpleDateFormat format = new SimpleDateFormat(ISO_DATE_FORMAT);
+        try {
+            standardSession.setCreationTime(format.parse(((String) metadata.get(METADATA_CREATION_TIME))).getTime());
+        } catch (ParseException e) {
+            e.printStackTrace();
```

Comment:
This is incredibly bad code, and I would reject the pull request for this alone. Don't ever check in code with `e.printStackTrace()`. Even in a `main` method, you shouldn't do it.